source/about/maximize-microsoft-investments.rst

```diff
@@ -0,0 +1,72 @@
+
+Mattermost complements Microsoft solutions with tailored capabilities designed to meet the unique needs of high-security environments, inter-agency collaboration, and external workflows, enabling teams to maximize their Microsoft investment.
+
+The following mission-ready collaboration capabilities are available:
+
+On-Premises Skype for Business Replacement
+-------------------------------------------
+
+As Skype for Business reaches end-of-life, secure organizations require an alternative that preserves on-premises control, integrates into Microsoft workflows, and meets the security standards of air-gapped and classified environments. Cloud-first replacements like Microsoft Teams are not always viable due to network segmentation, compliance restrictions, or data sovereignty mandates.
+
+- **Preserve mission-critical communication workflows** with a self-hosted Mattermost deployment that supports :doc:`1:1 calls </collaborate/make-calls>`, :ref:`screen sharing <collaborate/make-calls:share your screen>`, and :doc:`threaded messaging </collaborate/organize-conversations>` within secure environments. See :doc:`Mattermost for M365, Teams, and Outlook </integrate/mattermost-mission-collaboration-for-m365>`.
+- **Integrate Mattermost with Microsoft tools** such as Outlook, Teams, and :doc:`Entra ID Single Sign-On </onboard/sso-entraid>` to retain user workflows while centralizing identity and access control.
+- **Deploy in sovereign, air-gapped, or private cloud environments** such as `Azure Deployment <https://azuremarketplace.microsoft.com/en-us/marketplace/apps/mattermost.mattermost-operator?tab=overview>`_ to maintain compliance with STIG, FedRAMP, and NIST 800-53 standards.
+
+:doc:`Learn more </about/on-prem-skype-for-business-replacement>` about replacing Skype for Business with Mattermost.
+
+Out-of-Band Incident Response for Microsoft-Centric Environments
+-----------------------------------------------------------------
+
+During high-stakes incidents, Microsoft 365 tools can be limited or unavailable, slowing down response times and jeopardizing mission continuity.
+
+**Benefits**
+
+- **Maintain operational continuity during M365 outages** with a dedicated, out-of-band Mattermost instance for secure incident response, communication, and collaboration. See :doc:`Playbooks in Microsoft Teams </integrate/playbooks-for-microsoft-teams>`
+- **Accelerate responses** with :ref:`AI-powered workflows <configure/enable-copilot:mattermost configuration>`, enabling structured playbooks for triage, escalation, and resolution even when primary systems are compromised.
+- **Integrate with Microsoft Security Suite** and :doc:`Entra ID </onboard/sso-entraid>` to preserve centralized identity management while keeping sensitive data in a secure secondary system.
```

Comment:
Should we add a link to the integration capabilities like webhooks & slash commands?